src/responder/common/responder.h

```diff
@@ -108,6 +108,10 @@ struct resp_ctx {
     int domains_timeout;
     int client_idle_timeout;
 
+    time_t last_request_time;
```

Comment:
I wonder how does the responder idle shutdown work with the IFP responder, if it does? I set a quite short timeout in my testing for IFP and it seems to stay up forever..